gno.land/pkg/sdk/vm/keeper.go

```diff
@@ -46,7 +47,7 @@ const (
 type VMKeeperI interface {
 	AddPackage(ctx sdk.Context, msg MsgAddPackage) error
 	Call(ctx sdk.Context, msg MsgCall) (res string, err error)
-	QueryEval(ctx sdk.Context, pkgPath string, expr string) (res string, err error)
+	Eval(ctx sdk.Context, msg MsgEval) (res string, err error)
```

Comment:
```suggestion
	QueryEval(ctx sdk.Context, pkgPath, expr string, format Format) (res string, err error)
```

What about this?

Reply:
```suggestion
	QueryEval(ctx sdk.Context, cfg EvalCfg) (res string, err error)
```

To support optional flags, this is probably better.

Reply:
Agreed; if we have MsgEval it sounds like a transaction type, while it isn't.